SimPEG/electromagnetics/base_1d.py

```diff
@@ -17,9 +17,66 @@
     validate_integer,
 )
 from .. import props
+from ..maps import Wires
 from empymod.utils import check_hankel
 
-__all__ = ["BaseEM1DSimulation"]
+__all__ = [
+    "BaseEM1DSimulation",
+    "run_em1d_simulation",
+    "OutputType",
+    "ColeColeParameters",
+    "HankelCoefficients",
+]
+
+
+class OutputType(Enum):
+    FORWARD = 1
+    SENSITIVITY = 2
+
+
+@dataclass
+class HankelCoefficients:
+    """
+    Set of Hankel coefficients for a single sounding,
+
+    Arrays of shape(n_coefficients, n_frequency)
+
+    :param C0s: Coefficients for the zeroth order Hankel transform,
+    :param C1s: Coefficients for the first order Hankel transform
+    :param lambs: Array of Hankel transform parameters
+    :param unique_lambs: Unique Hankel transform parameters
+    :param inv_lambs: Inverse map of unique Hankel transform parameters
+    :param W: Array of weights
+    """
+
+    C0s: np.ndarray
+    C1s: np.ndarray
+    lambs: np.ndarray
+    unique_lambs: np.ndarray
+    inv_lambs: np.ndarray
+    W: sp.csr_matrix
+
+
+@dataclass
+class ColeColeParameters:
+    """
+    Set of static physical properties provided as an array of fixed values.
+
+    :param eta: Intrinsic chargeability
+    :param tau: Cole-Cole time constant
+    :param c: Cole-Cole frequency dependency
+    :param chi: DC magnetic susceptibility
+    :param tau1: Lower bound for log-uniform distribution of time-relaxation constants
+    :param tau2: Upper bound for log-uniform distribution of time-relaxation constants
+    """
+
+    eta: Union[np.ndarray, float] = 0.0
+    tau: Union[np.ndarray, float] = 1.0
+    c: Union[np.ndarray, float] = 0.5
+    chi: Union[np.ndarray, float] = 0.0
```

Comment:
@sgkang I found a few places where we had `dchi` and `chi`. I am guessing they are different, and would have different defaults. Can you validate this?